spec/components/home_page_hero_image_component_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe HomePageHeroImageComponent, type: :component do
+  let(:instance) { HomePageHeroImageComponent.new(override: override) }
+  let(:all_metadata) { instance.class.all_images_metadata }
+  let(:override) { nil }
+
+  describe "smoke test" do
+    it "renders" do
+      expect(render_inline(instance)).to have_selector('.hero-image')
+    end
+  end
+
+  describe "images in Rails.application.assets" do
+    it "all needed keys and images exist" do
+      expected_keys = ["link_title", "original_file", "path_1x", "path_2x", "work_friendlier_id"]
+      all_metadata.each do |metadata|
+        expected_keys.each do |k|
+          expect(metadata[k].class).to eq String
+          expect(Rails.application.assets.find_asset(metadata['path_1x']).class).to eq Sprockets::Asset
+          expect(Rails.application.assets.find_asset(metadata['path_2x']).class).to eq Sprockets::Asset
+        end
```

Comment:
I'm not sure about this Rails.applicaiton.assets.find_asset -- is that really public API?  OK, fine!  

I'm a bit worried this will be fragile, but I'm glad it's only in tests now, we'll find out!

I wonder if another way to do this would be getting access to ordinary Rails helper methods in tests, and just doing `image_path(thing)`, and making sure it doens't raise. 

Just musings, I am not requiring/requesting a change here. 

Nice test coverage generally! I like the idea of verifying this file in tests.